flutter/realm_flutter/tests/test_driver/app.dart

```diff
@@ -1,32 +1,31 @@
-import 'package:flutter_driver/driver_extension.dart';
+import 'dart:io';
 
+import 'package:flutter_driver/driver_extension.dart';
 import 'realm_test.dart' as tests;
 import 'const.dart';
 
 void main(List<String> args) async {
   enableFlutterDriverExtension(handler: (command) async {
     if (command!.startsWith(testCommand)) {
-
-      //Using the environment since --dart-entrypoint-args= is not working in 
-      //Flutter 2.5 with command 
+      //Using the environment since --dart-entrypoint-args= is not working in
+      //Flutter 2.5 with command
       //flutter drive -t test_driver/app.dart --dart-entrypoint-args="--testname"
       var testName = const String.fromEnvironment(envTestName);
       if (testName.isEmpty && args.isNotEmpty) {
         print("Using testName: \"${args[0]}\" from args");
         testName = args[0];
       }
-
-      List<String> testArgs = List<String>.empty();
+      List<String> testArgs = [];
       if (testName.isNotEmpty) {
         // Build correct test arguments using the dart test arg name '--name'
-        testArgs = ['--name', testName];
+        testArgs.addAll(['--name', testName]);
       }
+      testArgs.addAll(command.substring(command.indexOf("--")).split(" "));
```

Comment:
Did you test this with environment where `BASS_` env vars are not defined. Here `indexOf` can return -1. 

We should be able to run the tests locally with no `BAAS_` env vars.